docs/CHANGELOG.md

```diff
@@ -70,7 +70,8 @@ The sandbox cluster installation is running under the constant load generated by
 * BUGFIX: `vmstorage`: properly expire `storage/prefetchedMetricIDs` cache. Previously this cache was never expired, so it could grow big under [high churn rate](https://docs.victoriametrics.com/FAQ.html#what-is-high-churn-rate). This could result in increasing CPU load over time.
 * BUGFIX: [vmalert](https://docs.victoriametrics.com/vmalert.html): check `-external.url` schema when starting vmalert, must be `http` or `https`. Before, alertmanager could reject alert notifications if `-external.url` contained no or wrong schema.
 * BUGFIX: [vmalert](https://docs.victoriametrics.com/vmalert.html): automatically add `exported_` prefix for original evaluation result label if it's conflicted with external or reserved one, previously it was overridden. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/5161).
-* BUGFIX: [vmalert](https://docs.victoriametrics.com/vmalert.html): autogenerate `ALERTS_FOR_STATE` time series for alerting rules with `for: 0`. Previously, `ALERTS_FOR_STATE` was generated only for alerts with `for > 0`. The change aligns with Prometheus behavior. See more details in [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/5648). 
+* BUGFIX: [vmalert](https://docs.victoriametrics.com/vmalert.html): autogenerate `ALERTS_FOR_STATE` time series for alerting rules with `for: 0`. Previously, `ALERTS_FOR_STATE` was generated only for alerts with `for > 0`. The change aligns with Prometheus behavior. See more details in [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/5648).
+* BUGFIX: [vmalert](https://docs.victoriametrics.com/vmalert.html): set `ActiveAt` timestamp of created alerts to the execution timestamp instead of the actual timestamp when rule was executed. The change is justified by [alert generator specification](https://github.com/prometheus/compliance/blob/main/alert_generator/specification.md#pending-state-conditions): "evaluation time when the alert is first created is referred to as ActiveAt for that alert".
```

Comment:
There is no bug now but duplicated assignment to `ActiveAt` as commented [here](https://github.com/VictoriaMetrics/VictoriaMetrics/pull/5657/files#r1463005474)

Reply:
Oh, right. I misunderstood the previous comment about "bug". You're right! See updated